presto-common/src/main/java/com/facebook/presto/common/block/EncoderUtil.java

```diff
@@ -97,4 +100,41 @@ public static void encodeNullsAsBits(SliceOutput sliceOutput, Block block)
 
         return Optional.of(valueIsNull);
     }
+
+    /**
+     * Decode the bit stream created by encodeNullsAsBits.  The valueIsNull buffer must be exactly sized to the position
+     * count.  Returns the number of nulls seen during deserialization.
+     */
+    public static int decodeNullBits(SliceInput sliceInput, boolean[] valueIsNull)
```

Comment:
This new method shares 90% same code as existing `public static Optional<boolean[]> decodeNullBits(SliceInput sliceInput, int positionCount)`. Can we extract the common part into a method to reduce code duplication?

Reply:
Emm, I see the new method is doing bitCount on the fly. So it makes sense. Please disregard this comment.